src/store.coffee

```diff
@@ -16,7 +16,8 @@ module.exports = (session) ->
 		###
 		
 		constructor: (config, options) ->
-			@table = options.table if options?.table
+			@table = "[" + options.table + "]" if options?.table
+			@table = "[" + config.schema + "].#{@table}" if config?.schema
```

Comment:
I don't think `config` is a good collection to specify `schema` in. Why not to let user specify schema inside `table` variable together with the table name?